src/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom';
+
+export const Header = () => {
+  return (
+    <div className="App">
+      <h1>People table</h1>
+      <nav className="nav">
+        <Link
```

Comment:
You didn't add link to /home
<img width="372" alt="image" src="https://user-images.githubusercontent.com/30314756/176430408-3fa4a4c7-b9b5-4c6d-a17b-c00e248ed9b5.png">